ci/build-mingw64.sh

```diff
@@ -141,7 +148,7 @@ _ffmpeg () {
         --pkg-config=pkg-config --target-os=mingw32
         --enable-cross-compile --cross-prefix=$TARGET- --arch=${TARGET%%-*}
         --cc="$CC" --cxx="$CXX" $commonflags
-        --disable-{doc,programs,muxers,encoders}
+        --disable-{doc,programs}
```

Comment:
I'd rather only enable one relevant muxer or encoder for the tests (as on the line below)

Reply:
What if user wants to use different format? What is the gain to exclude some of them?

Reply:
But what is the gain of excluding what is available?

Reply:
Lower compile times.

Reply:
I see no significant difference. In cached state it is 4s difference, clean build was also not visibly slower, but I can test that to confirm.

Reply:
This excludes more than 200 files so it should definitely make a difference uncached.

Reply:
Execution times of `Build libraries` step
|                  | master | #14207 |       |
|------------------|:------:|--------|-------|
| i386 \| cached   |  210 s |  170 s |  80 % |
| i386 \| clean    |  976 s | 1026 s | 105 % |
| x86_64 \| cached |  262 s |  247 s |  94 % |
| x86_64 \| clean  | 1083 s | 1162 s | 107 % |

It does, but the difference in my opinion is not significant enough to making some arbitrary list of enabled/disabled muxers. Note that we always never do full rebuild, I needed to manually remove cache from my fork to test that.

Tested on CI with the following builds:
https://github.com/mpv-player/mpv/actions/runs/9191373753/job/25277639660
https://github.com/mpv-player/mpv/actions/runs/9183181995/job/25253412400
https://github.com/kasper93/mpv/actions/runs/9192050397/job/25279896478
https://github.com/kasper93/mpv/actions/runs/9192086594/job/25279911198

Reply:
>Most important encoders like x264/x265 aren't ffmpeg built-in anyway

At least you could save png/jpg screenshot...